### PR TITLE
Add missing steps on how to use the -Ounchecked flag

### PR DESCRIPTION
**Pull request by own2pwn**
This PR adds missing steps on how to pass the `-Ounchecked` flag.

### CODE REVIEW

**Comment by own2pwn** [comment]
Project
whole-project-unchecked

Target
target-unchecked

**Comment by own2pwn** [comment]
@swift-ci Please smoke test

File: docs/OptimizationTips.rst [hunk 1]
```diff
@@ -40,7 +40,10 @@ four different optimization levels:
 
 In the Xcode UI, one can modify the current optimization level as follows:
 
-...
+For the whole project,
+Select project in `project and targets list`, then add `-Ounchecked` in `Other Swift Flags` under `Swift Compiler - Custom Flags`.
```
Comment (CodaFi) [inline docs/OptimizationTips.rst]: Nit: It seems like this wants a section on attaching *any* of the flags mentioned above to a compile command line.  Perhaps

```
In the Project Navigator, select the Project to enter the Project Editor. From there, an optimization setting can be applied to every target in the project by changing the "Optimization Level" field under the `Build Settings` header.

To apply a custom optimization level to a singular target, select that target in the Project Editor and override the "Optimization Level" field under its `Build Settings` header.

If a given optimization level is not available in the UI, its corresponding flag can be specified by selecting the `Other...` level in the "Optimization Level" dropdown.
```

**Comment by CodaFi** [comment]
@own2pwn Thanks for improving this.  I've left some suggestions for wording this section.

Also, CI is unavailable to those without the write bit for security purposes.

**Comment by own2pwn** [comment]
My bad, I've been searching about optimization options and didn't notice this sections tells about all possible options. I don't mid if we'll user your version.

**Comment by futurejones** [comment]
Can one of the admins verify this patch?

**Comment by CodaFi** [comment]
Merged the linked PR.

**Pull request closed without merging**